src/phoenix/db/migrations/versions/3be8647b87d8_add_token_columns_to_spans_table.py

```diff
@@ -111,24 +110,17 @@ class Span(Base):
 LLM_TOKEN_COUNT_COMPLETION = SpanAttributes.LLM_TOKEN_COUNT_COMPLETION.split(".")
 
 
-async def get_token_counts_from_attributes(connection: AsyncConnection) -> None:
-    """
-    Gets token counts from attributes if present.
-    """
-    await connection.execute(
+def upgrade() -> None:
+    op.add_column("spans", sa.Column("llm_token_count_prompt", sa.Integer, nullable=True))
+    op.add_column("spans", sa.Column("llm_token_count_completion", sa.Integer, nullable=True))
+    op.execute(
```

Comment:
did you mean to touch this file?

Reply:
It's no longer an async engine, and also, since we are waiting for the thread to join, async is not necessary.